Tests/HyperwalletTests.swift

```diff
@@ -68,6 +68,7 @@ class HyperwalletTests: XCTestCase {
             error: authErrorResponse)
 
         // When
+        Hyperwallet.clearInstance()
```

Comment:
Instead of adding clearInstance in existing tests, create new func testClearInstance and then call setup 
```suggestion
        func testClearInstance() {
        Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
        let hyperwalletInstance1 = Hyperwallet.shared
        XCTAssertNotNil(hyperwalletInstance1)
        Hyperwallet.clearInstance()
        Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
        let hyperwalletInstance2 = Hyperwallet.shared
        XCTAssertNotNil(hyperwalletInstance2)
        XCTAssertNotEqual(hyperwalletInstance1, hyperwalletInstance2, "hyperwalletInstance2 should not be same as hyperwalletInstance1")
    }
```

Reply:
I have added the testClearInstance method. But if we remove  Hyperwallet.clearInstance() the testSetup_getConfiguration_authenticationError and testSetup_getConfiguration_decodeError fails because ,since only one instance are created while exeucting this error testcase gets fail.(Getting success respond)